styx-scheduler-service/src/main/java/com/spotify/styx/state/handlers/FlyteRunnerHandler.java

```diff
@@ -108,18 +106,12 @@ private void transitionSubmitting(RunState state, EventRouter eventRouter) {
     sendEventIgnoreClosed(state, eventRouter, "submitted", submitted);
   }
 
-  private void transitionSubmitted(RunState state, EventRouter eventRouter) {
-    LOG.info("Entered state SUBMITTED for: " + state.workflowInstance());
-    final var started = Event.started(state.workflowInstance());
-    LOG.info("Issue 'started' event for: " + state.workflowInstance());
-    sendEventIgnoreClosed(state, eventRouter, "started", started);
-  }
-
-  private void transitionRunning(RunState state, EventRouter eventRouter) {
-    LOG.info("Entered state RUNNING for: " + state.workflowInstance());
+  private void pollingExecution(RunState state, EventRouter eventRouter) {
+    LOG.info("Entered state" + state.state().toString() + " for: " + state.workflowInstance());
     final FlyteExecConf flyteExecConf = state.data().executionDescription().orElseThrow().flyteExecConf().orElseThrow();
```

Comment:
I know it is not the problem of this PR, but this method does too much and leaks `FlyteExecConf` that we can probably expose to flyte runner only, plus `flyteRunner.poll` needs `state` anyway.

Reply:
You can ignore this comment as I see the rest of the class does things similar.